src/components/GoogleMapContainer.ts

```diff
@@ -127,14 +133,25 @@ class GoogleMapContainer extends Component<GoogleMapContainerProps, { alertMessa
     }
 
     // Mendix does not support negative and decimal number as static inputs, so they are strings.
-    public static parseStaticLocations(locations: StaticLocation[]): Location[] {
-        return locations.map(location => ({
+    public static parseStaticLocations(props: GoogleMapContainerProps): Location[] {
+        return props.staticLocations.map(location => ({
             address: location.address,
             latitude: location.latitude.trim() !== "" ? Number(location.latitude) : undefined,
-            longitude: location.longitude.trim() !== "" ? Number(location.longitude) : undefined
+            longitude: location.longitude.trim() !== "" ? Number(location.longitude) : undefined,
+            url:  GoogleMapContainer.getStaticMarkerUrl(location, props.defaultMakerIcon)
         }));
     }
 
+    private static getStaticMarkerUrl(location: StaticLocation, defaultMakerIcon: string): string {
+        if (location.icon) {
+            return UrlHelper.getStaticResourceUrl(location.icon);
+        }else if (defaultMakerIcon) {
+            return UrlHelper.getStaticResourceUrl(defaultMakerIcon);
+        }else {
+            return "";
```

Comment:
Would this 3rd options fallback to the default marker that is shipped with the widget?